mock-omaha-server/Cargo.toml

```diff
@@ -18,6 +18,11 @@ publish = false
 
 exclude = [".*"]
 
+[features]
```

Comment:
If a user is getting this from crates.io, they won't have the ability to use the `fasync` feature since we haven't published the `fuchsia-async` crate. How about we use a hidden feature flag for it following from https://blog.rust-lang.org/2024/05/06/check-cfg.html. I think it'd look something like:

```
[features]
default = ["dep:tokio"]

[dependencies]
...
tokio = { version = "1", features = ["full"], optional = true }
...

[lints.rust]
unexpected_cfgs = { level = "warn", check-cfg = ['cfg(fasync)'] }
```

Then in fuchsia's https://cs.opensource.google/fuchsia/fuchsia/+/main:third_party/rust_crates/Cargo.toml we would import it as:

```
mock-omaha-server = { version = "...", default-features = false }
...
[gn.package.mock-omaha-server."..."]
rustflags = [
  "--cfg=fasync",
]

```

Reply:
Done.